Source/Common/GLTFUnarchiver.swift

```diff
@@ -1634,7 +1633,7 @@ public class GLTFUnarchiver {
             scnNode.position = createVector3(glNode.translation)
         }
         
-        if let weights = glNode.weights {
+        if glNode.weights != nil {
```

Comment:
Can switch this back to the previous syntax once weights loading is implemented.